GovUkDesignSystem/GovUkDesignSystemComponents/SubComponents/SummaryListRowAction.cshtml

```diff
@@ -1,18 +1,36 @@
 ﻿@model GovUkDesignSystem.GovUkDesignSystemComponents.SummaryListRowActionViewModel
 
 <dd class="govuk-summary-list__actions @Model?.Classes">
-    @{
-        if (Model != null && Model.Items != null && Model.Items.Count > 0)
+    @if (Model?.Items != null && Model.Items.Count > 0)
+    {
+        if (Model.Items.Count == 1)
```

Comment:
Why do we need a special case for 1 action? Will things align nicely if one row has one action and the next row has multiple?

Reply:
In one case we don't want to put the action in a ul and li, again going off the govuk njk (https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/summary-list/template.njk)

Reply:
Fair enough, if the GovUK control does this then we should too